Formula/zf.rb

```diff
@@ -1,8 +1,9 @@
 class Zf < Formula
   desc "Command-line fuzzy finder that prioritizes matches on filenames"
   homepage "https://github.com/natecraddock/zf"
-  url "https://github.com/natecraddock/zf/archive/refs/tags/0.6.0.tar.gz"
-  sha256 "9767ce1142933c753b749630229d10519c4c0b208f63931e43495acc632135e2"
+  url "https://github.com/natecraddock/zf.git",
```

Comment:
can you add a code comment about when we can revert back to the tarball file? Thanks!

Reply:
Absolutely! I just updated the PR

Reply:
Thanks!